code/modules/clothing/under/accessories/badges.dm

```diff
@@ -244,42 +244,236 @@
+	contained_sprite = 1
+	slot_flags = null
+	w_class = ITEMSIZE_TINY
+	flippable = FALSE
+	v_flippable = FALSE
+
+/obj/item/clothing/accessory/badge/passcard_silversun
+	name = "silversun commemorative passcard"
+	desc = "A passcard issued to Idris employees currently or formerly employed or residing on the planet Silversun."
+	desc_fluff = "While Silversun itself is a member of the Sol Alliance, Idris Incorporated has secured a number of obscure patents, permits, and bureaucratic channels that allows them to commemorate \
+	faithful employees from the resort world with unique passcards that double as membership cards to some of the cheaper resorts on-world."
+	icon = 'icons/clothing/accessories/passcards.dmi'
+	icon_state = "passcard_silversun"
+	item_state = "passcard_silversun"
+	overlay_state = "passcard_silversun"
+	contained_sprite = 1
+	slot_flags = null
+	w_class = ITEMSIZE_TINY
+	flippable = FALSE
+	v_flippable = FALSE
+
+/obj/item/clothing/accessory/badge/passcard_eridani
+	name = "eridani corporate republic passcard"
+	desc = "A holographic passcard issued to residents of the Free Economic Zone of Epsilon Eridani."
+	desc_fluff = "Known with a number of unfavorable acronyms across the Republic, Eridani passcards tend to include unhelpful details such as credit score, personal debts, and insurance providers to those \
+	equipped with the right equipment. Infamous across the Orion Spur for being the most sought-after passcards for counterfeiting."
+	icon = 'icons/clothing/accessories/passcards.dmi'
+	icon_state = "passcard_eridani"
+	item_state = "passcard_eridani"
+	overlay_state = "passcard_eridani"
+	contained_sprite = 1
+	slot_flags = null
+	w_class = ITEMSIZE_TINY
+	flippable = FALSE
+	v_flippable = FALSE
+
+/obj/item/clothing/accessory/badge/passcard_elyra
+	name = "elyran passcard"
+	desc = "A passcard issued to citizens of the Serene Republic of Elyra."
+	desc_fluff = "Much like synthetics manufactured in the Serene Republic, Elyran passcards include high-grade anti-counterfeiting through a wafer of borosilicate reinforced with a randomly-chosen alloy. \
+	Unlike other nations, however, Elyra allows for heavy personal customization of their passcards, with some of the most expensive government contractors offering simple musical notes or integration with \
+	jewelry such as a bracelet or necklace."
+	icon = 'icons/clothing/accessories/passcards.dmi'
+	icon_state = "passcard_elyra"
+	item_state = "passcard_elyra"
+	overlay_state = "passcard_elyra"
+	contained_sprite = 1
+	slot_flags = null
+	w_class = ITEMSIZE_TINY
+	flippable = FALSE
+	v_flippable = FALSE
+
+/obj/item/clothing/accessory/badge/passcard_dominia
+	name = "dominian passcard"
+	desc = "A passcard issued to citizens of the Empire of Dominia."
+	desc_fluff = "Dominian passcards, aside from the usual information, also include details such as ones House, their remaining Mo'ri'zal, their status as an Edict Breaker, or- most uniquely- their most recent \
```

Comment:
Should be "one's house" rather than "ones house" I imagine.

Reply:
got you gamer 😎